migration/migration-web-developers.md

```diff
@@ -0,0 +1,251 @@
+```
+
+#### Now (PoS)
+
+```JavaScript
+// No equivalent yet
+```
+
+### Export the key pair or private key as a plain Uint8Array
+To export the key pair or private key as a plain Uint8Array, you need to serialize the key pair or private key.
+
+#### Previous (PoW)
+
+```JavaScript
+const keypairBytes = wallet.exportPlain(); // [privatekey, publickey]
+const privatekeyBytes = wallet.keyPair.privateKey.serialize();
+```
+
+#### Now (PoS)
+
+```JavaScript
+const keypairBytes = keyPair.serialize(); // [publickey, privatekey] - to be reversed in 0.22.0
+const privatekeyBytes = keyPair.privateKey.serialize();
+```
+
+### Export the private key as an encrypted Uint8Array
+Previously, you could export the private key as an encrypted Uint8Array with the ```Wallet.exportEncrypted()``` function. This function is no longer available.
+
+#### Previous (PoW)
+
+```JavaScript
+const encryptedKey = await wallet.exportEncrypted(password);
+```
+
+#### Now (PoS)
+
+```JavaScript
+// No equivalent yet
+```
+
+### Create and send transactions
+In order to send a transaction, you need to create a transaction builder and sign it with the key pair.
+
+#### Previous (PoW)
+
+```JavaScript
+const transaction = wallet.createTransaction(
+    Nimiq.Address.fromString(recipient),
+    Nimiq.Policy.coinsToLunas(amount), // Convert from NIM to luna
+    0, // Fee, optional
+    await client.getHeadHeight() // Current blockchain height
+);
+
+const txDetails = await client.sendTransaction(transaction);
+```
+
+#### Now (PoS)
+
+```JavaScript
+const transaction = Nimiq.TransactionBuilder.newBasic(
+    keyPair.toAddress(),
+    Nimiq.Address.fromString(recipient),
+    BigInt(amount * 1e5), // Convert from NIM to luna
+    BigInt(0), // Fee, optional
+    await client.getHeadHeight(),
+    await client.getNetworkId(),
+);
+transaction.sign(keyPair);
+
+const txDetails = await client.sendTransaction(transaction);
+```
+
+### Convert keys
+Convert the public and private keys to their hexadecimal string representations.
+
+#### Previous (PoW)
+
+```JavaScript
+const publickeyHex = wallet.publicKey.toHex(); // or wallet.keyPair.publicKey.toHex()
+const privatekeyHex = wallet.keyPair.privateKey.toHex();
+```
+
+#### Now (PoS)
+
+```JavaScript
+const publickeyHex = keyPair.publicKey.toHex();
+const privatekeyHex = keyPair.privateKey.toHex();
+```
+
+## Nimiq.Client
+The Client class is a central component of the Nimiq's new web client framework and is essential for interacting with the Nimiq blockchain network under the PoS consensus model.
+
+### Initialization
+
+#### Previous (PoW)
+
+```JavaScript
+import Nimiq from '@nimiq/core-web';
+
+await Nimiq.init();
+Nimiq.GenesisConfig.test(); // Select testnet
+const configBuilder = Nimiq.Client.Configuration.builder();
+const client = configBuilder.instantiateClient();
+```
+
+#### Now (PoS)
+
+When using the /web package export, you need to manually call the init function:
```

Comment:
There are slight differences between webpack and vite. What about adding a link to the integrations?